### PR TITLE
examples - make examples work

### PR DESCRIPTION
**Pull request by danielcompton**
I tried to run the example and couldn't get it to work. I started fixing it, and wondered whether it might be helpful for others as well.

Feel free to make any improvements you might have, I'm completely new to Bazel, but very interested in using it at work.

I'm also not sure why CI isn't running on this PR.

### CODE REVIEW

File: examples/setup/default/example_library/BUILD [hunk 1]
```diff
@@ -0,0 +1,8 @@
+load("@rules_clojure//:rules.bzl", "clojure_library")
+
+clojure_library(
+    name = "lib1",
+    srcs = ["src/example/core.clj"],
+    aot = ["example.core"],
+    resource_strip_prefix = "example_library/src",
```
Comment (danielcompton) [inline examples/setup/default/example_library/BUILD:7]: I'm not sure if I'm misconfiguring this, but I needed to include `example_library/` for this to work.

File: examples/setup/default/WORKSPACE [hunk 2]
```diff
@@ -1,5 +1,20 @@
 workspace(name = "default")
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```
Comment (danielcompton) [inline examples/setup/default/WORKSPACE:3]: When I was running the build, Bazel complained about skylib being missing. https://stackoverflow.com/a/65682139 suggested that I needed to add skylib, though I'm not confident I've done this correctly.

**Comment by arohner** [comment]
Sorry about that. I've made numerous fixes in #48 which should be merged relatively soon.

**Comment by arohner** [comment]
I've also configured CI to run against forks

**Pull request closed without merging**